SmartDeviceLink/SDLCarWindow.m

```diff
@@ -120,13 +121,18 @@ - (void)sdl_didReceiveVideoStreamStarted:(NSNotification *)notification {
 
     dispatch_async(dispatch_get_main_queue(), ^{
         // If the video stream has started, we want to resize the streamingViewController to the size from the RegisterAppInterface
-        self.rootViewController.view.frame = CGRectMake(0, 0, self.streamManager.screenSize.width, self.streamManager.screenSize.height);
+        self.rootViewController.view.frame = self.sdl_getScaledScreenSizeFrame;
         self.rootViewController.view.bounds = self.rootViewController.view.frame;
-
+        
         SDLLogD(@"Video stream started, setting CarWindow frame to: %@", NSStringFromCGRect(self.rootViewController.view.bounds));
     });
 }
 
+- (CGRect)sdl_getScaledScreenSizeFrame {
```

Comment:
Misleading method name:
1. We can use a property name without the prefix get
- (CGRect)sdl_scaledScreenSizeFrame {}
2. If we really want to use the prefix get then the method must look like this:
- (void)sdl_getScaledScreenSizeFrame:(CGRect * outRect) {
    if (outRect) {
       *outRect = <the rect value>;
    }
}